include/deal.II/matrix_free/dof_info.templates.h

```diff
@@ -497,9 +498,14 @@ namespace internal
         const_cast<Utilities::MPI::Partitioner *>(vector_partitioner.get());
       vec_part->set_ghost_indices(ghost_indices);
 
-      vector_exchanger = std::make_shared<
-        internal::MatrixFreeFunctions::VectorDataExchange::PartitionerWrapper>(
-        vector_partitioner);
+      if (false)
+        vector_exchanger =
+          std::make_shared<internal::MatrixFreeFunctions::VectorDataExchange::
+                             PartitionerWrapper>(vector_partitioner);
```

Comment:
Is this intentional that you use this exchange scheme already now? I assumed from the PR description that it is not of much use (but correct) and thus enabled later? Not that I'm opposed to it, but this looks more like a performance setup to be tested rather than the default before the full functionality is available.

Reply:
Would it be possible for the time being to introduce a new parameter in `AdditionalData`. Hopefully, we can remove it sometime!

Reply:
I guess you intend to look into both schemes, so I would be fine with having that parameter for the time being; if you feel it can also be done with the present `if` statements in the code I would also be fine with the current version.

Reply:
> I guess you intend to look into both schemes, so I would be fine with having that parameter for the time being; if you feel it can also be done with the present if statements in the code I would also be fine with the current version.

My final goal is to through out the usage of `Partitioner` inside of `MatrixFree`. But for the time being (as long as we don't know if the new class performs at least as good as the `Partitioner`), I would like to have an easy way to switch between implementations.

Reply:
Sounds good, let's go with this temporary solution.